src/main/java/com/synopsys/integration/alert/workflow/processor/MessageContentAggregator.java

```diff
@@ -50,6 +53,7 @@
 
 @Component
 public class MessageContentAggregator {
+    private static final Logger logger = LoggerFactory.getLogger(MessageContentAggregator.class);
```

Comment:
It looks like we needed a logger then never actually used it.

Reply:
It was used earlier for debugging.  I will remove it.